src/AcceptanceTests/Routing/When_sending_to_specific_namespace.cs

```diff
@@ -103,6 +103,46 @@ public class When_using_multiple_namespaces : NServiceBusAcceptanceTest
             Assert.IsTrue(context.ReplyReceived, "context.ReplyReceived");
         }
 
+        [Test]
+        public async Task Should_support_request_reply_across_namespaces_using_contained_endpoint()
+        {
+            var runSettings = new RunSettings();
+            runSettings.TestExecutionTimeout = TimeSpan.FromMinutes(1);
+
+            var connectionString = Environment.GetEnvironmentVariable("AzureServiceBusTransport.ConnectionString");
+            var targetConnectionString = Environment.GetEnvironmentVariable("AzureServiceBus.ConnectionString.Fallback");
+
+            var ctx = new AzureServiceBusTransportConfigContext();
+            ctx.Callback = (endpointName, extensions) =>
+            {
+                if (endpointName == Conventions.EndpointNamingConvention(typeof(EndpointInTargetNamespace)))
```

Comment:
Personally I would flip if/else to emphasize that we're testing sending (routing) outside of the source namespace via `NamespaceRouting()` API. Not critical though.

Reply:
I kept it aligned with the other tests. I think I'll leave it